### PR TITLE
 collaborate branch

### PR DESCRIPTION
**Pull request by CELYNEBIRUNGI**
- No issues found at page titles
- Image text Alts added  https://github.com/CELYNEBIRUNGI/Portfolio-Project/blob/b179aa802035f522bd662df66835274e59d1ca9b/index.html#L29-L33
- https://github.com/CELYNEBIRUNGI/Portfolio-Project/blob/b179aa802035f522bd662df66835274e59d1ca9b/index.html#L68
- Text heading no issues found
- No contrast issues found.
- No sizing issues found
- No basic structure issues found
- No moving content issues found
- No multi-media issues found
- No interaction issues found

### CODE REVIEW

File: index.html
Comment (AbelG101) [inline index.html:1]: - Great job determining changes in your Portfolio 👏 however you need to make some changes to fix the accessibility issues as requested in the project requirement.
    - [x] I noticed that the social media icons in the "home page" and the "about section" don't have `alt` texts kindly make sure that all `img` tags have `alt` attribute with an appropriate alternative text, it is used by people who have visual impairments and use screen readers to hear the alt text read out.
 - [x] As specified in the project requirements please make sure to link to the corresponding line of code ([here's how](http://www.chasewoodford.com/blog/linking-highlighted-lines-code-github/#:~:text=To%20link%20to%20multiple%20lines,php%23L117%2DL148)) where the changes are required in your PR description. 
     firefox_Qld0tf4cYR
- [x] Once again as specified in the project requirements your report should list all points that you have checked you seem to be missing the below checks so please include them:
    - Moving content ❌
    - Multimedia ❌
    - Interaction ❌

**Review by AbelG101 — CHANGES_REQUESTED** [review]
# Hi @zbahati 👋, 

You're doing a good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- Correct use of GitHub Flow ✔
-  Well-structured files ✔
- All linter checks are passing ✔
- Descriptive PR title with a great summary ✔

## Required Changes ♻️
- _Check the comments under the review._
- You just have to make a few changes and you're on your way 🚀.


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

You can also consider:
- N/A

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md
Comment (AbelG101) [inline README.md:1]: - [x] As I requested previously please make sure to link to the corresponding line of code/s in the report where you found some issues in the code. [You can refer to this link for guidance](http://www.chasewoodford.com/blog/linking-highlighted-lines-code-github/#:~:text=To%20link%20to%20multiple%20lines,php%23L117%2DL148).
- [x] This is a [project requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/6_accessability.md) so please make sure you fix it in your next review:

  image

**Review by AbelG101 — CHANGES_REQUESTED** [review]
# Hi @CELYNEBIRUNGI & @husush 👋, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
- Correct use of GitHub Flow ✔
-  Well-structured files ✔
- All linter checks are passing ✔
- Descriptive PR title with a great summary ✔

 ## Required Changes ♻️
- _Check the comments under the review._
- Please go through the comments carefully before requesting a review as I noticed you forgot to implement one of the requirements.

## Optional suggestions
- N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md
Comment (husush23) [reply README.md:1]: https://github.com/CELYNEBIRUNGI/Portfolio-Project/blob/b179aa802035f522bd662df66835274e59d1ca9b/index.html#L29-L33

Comment (Shedrack-Sunday) [reply README.md:1]: - [ ] Ass requested by the previous reviewer, please make sure to link to the corresponding line of code/s in the report where you found some issues in the code. You included a snapshot, which does not pass the project requirements. 
-  [ ] You can refer to this [link for guidance](http://www.chasewoodford.com/blog/linking-highlighted-lines-code-github/#:~:text=To%20link%20to%20multiple%20lines,php%23L117%2DL148).

 - This is a [project requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/6_accessability.md) and its needed to get you approved 👍 
 
 ## Instructions 
 
image

**Review by Shedrack-Sunday — CHANGES_REQUESTED** [review]
Hi @CELYNEBIRUNGI            👋 

# CHANGES REQUIRED ♻️

Great work on the project 👏
You've done well implementing the requirements,  However, there are a few you need to work on, to move to the next project. 💯

 


## Highlights
- Correct use of GitHub Flow ✔
- Well-structured files ✔
- All linter checks are passing ✔
- Descriptive PR title with a great summary ✔
  
# Required Changes ♻️

Check the comments under the review.
You just have to make a few changes based on project requirements, and you're on your way 🚀. 

# Optional suggestions 

Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application.

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Comment by bimbolabuari** [comment]
Hello @CELYNEBIRUNGI,

Your project already mets all requirements, and your project will be approved manually.

**Review by wayungi — APPROVED** [review]
Hi @CELYNEBIRUNGI , 

image

Great work, your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

**Remember: Every finish line 🏁  is the beginning of a new race 🐎** 

## Highlights
- [ ] All accessibility checks are good ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**